### ISSUE TITLE
Blockchain: Building Public Trust in Government Data

### ISSUE THREAD

**Issue by philipashlock** [GSA/participate-nap4#15]

**Comment by jstclair-HFT** [comment]
I would be happy to assist.

**Comment by akarides** [comment]
- Disseminate government data on a decentralized blockchain, which is supported by the baked-in security and verifiability properties of the technology
- Public trust in the government is restored in two ways: 1) the sheer amount of additional public data & information that is available 2) the data is validated as trustworthy because it lives on a blockchain

**Comment by bsweger** [comment]
**TL;DR**
This is a big, hairy problem that almost certainly cannot be solved with technology.

Before suggesting a technical solution for improving public trust in government data, we have to understand _why_ the public doesn't trust government data.

For example:

1. Data is hard to find
2. Data is hard to use (it's in PDF, it's fronted by non-intuitive interfaces, etc.)
3. Data lacks documentation, making it impossible to understand key information like data provenance and last update
4. Data is published irregularly
5. Agencies don't provide public support for the data they publish, often due to lack of resources
6. There are multiple federal data sources for similar topics
7. It's difficult or impossible to link together related data
8. Some agencies are exempted from or ignore federal data quality standards
9. Data definitions and formats change from year to year, making time series analysis challenging or impossible
10. Public has no way of knowing what data _isn't_ published

(this list is anecdotal, drawn from personal experience in a private sector job)

Now that I work for the federal government (though am commenting here in a personal capacity), I've seen that many of the above grievances result from a culture that actively discourages sharing data and engaging with the public.

Technology is a critical tool, but it will never solve open data's political and cultural challenges. In fact, starting with a technical solution often diverts limited attention and resources away from the real hard work, resulting in a waste of tax dollars.

A better place to start might be to find out who doesn't trust government data and why. Only then will we have the necessary info to outline a possible solution to this important problem.

**Comment by JoshData** [comment]
I love that list @bsweger.

**Comment by akarides** [comment]
#Cocreation 

#Topline Description
Incentivizing Open Government Emerging Tech Innovation Initiative to Improve Emergency Response

#Measurable Metrics
•	Establish decentralized distributed open data on a blockchain containing critical emergency response data
o	Opening data on-ramps through building open API’s and open source software
o	Storing datasets in multiple distributed locations
o	Opening federally funded research data conducted by major research organizations, scientific publications, and professional associations to have their data published openly
•	Professionalize the Management of Data
o	Declaring emergency response data as a national asset
•	Encourage private-public collaboration to improve emergency response through the implementation of open procurement by offering relevant documents on a blockchain

#Outcomes 
• Agencies, departments, and organizations are incentivized to achieve these goals by receiving capital funds comprised of the money saved by enacting and these technologies (see cloud computing incentives in MGT Act)
•Lives are saved through enhanced emergency response
